### PR TITLE
Introduce a new `ps` command that displays containers

### PR DESCRIPTION
**Pull request by sroze**
This PR, needed by #8, introduce a new `ps` command that list all the running or exited containers of the application. The main thing is that it also display the containers' DNS.

Here's a screenshot of the current output:
screen shot 2015-07-06 at 23 20 45

### CODE REVIEW

File: src/Cli/PsCommand.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        $userInteraction = new ConsoleUserInteraction($input, $output);
+        $processRunner = new SilentProcessRunner($userInteraction);
+
+        $inspector = new Inspector($processRunner);
+        $containers = $inspector->getRunningContainers();
+
+        $table = new Table($output);
+        $table->setHeaders(['Name', 'DNS addresses', 'Port(s)', 'Status']);
+
+        foreach ($containers as $index => $container) {
```
Comment (richardmiller-zz) [inline src/Cli/PsCommand.php]: Is it worth extracting this part out of the command class as a view?

Comment (sroze) [inline src/Cli/PsCommand.php]: Yes, that makes sense.

**Comment by sroze** [comment]
@richardmiller moved the table display logic.

**Comment by richardmiller-zz** [comment]
:+1: Looks good to merge to me.

**Pull request merged**